src/rcc.rs

```diff
@@ -15,6 +15,8 @@ pub trait RccExt {
 }
 
 impl RccExt for RCC {
+
+    #[cfg(not(feature = "stm32l47x"))]
```

Comment:
I think it would be better to only feature gate specific fields, instead of the whole RccExt impl.

Reply:
crrcr does not exit for stm3247x  how do you see implementation of the constrain ?

Reply:
Using `#[cfg(not(feature = "stm32l47x"))]` on the fields instead of the entire impl, e.g

```
    fn constrain(self) -> Rcc {
        Rcc {
            ahb1: AHB1 { _0: () },
            ahb2: AHB2 { _0: () },
            ahb3: AHB3 { _0: () },
            apb1r1: APB1R1 { _0: () },
            apb1r2: APB1R2 { _0: () },
            apb2: APB2 { _0: () },
            bdcr: BDCR { _0: () },
            csr: CSR { _0: () },
            #[cfg(not(feature = "stm32l47x"))]
            crrcr: CRRCR { _0: () },
            cfgr: CFGR {
                hclk: None,
                usb48: false,
                lsi: false,
                pclk1: None,
                pclk2: None,
                sysclk: None,
                pllcfg: None,
            },
        }
    }
```